Makefile.defs

```diff
@@ -34,6 +34,11 @@ export CMAKE_BUILD_TYPE
 
 # CMAKE=cmake --no-warn-unused-cli
 CMAKE=cmake
+KO_BUILD_SYSTEM=$(MAKE)
+ifdef USE_NINJA
+	CMAKE:=cmake -G Ninja
+	KO_BUILD_SYSTEM:=ninja
```

Comment:
@NiLuJe Any feedback on the variable name?

Reply:
Not a huge fan, but I'm at a loss for anything better right now...

Reply:
(It's mainly the loss of "MAKE" in there that bothers me, as it's recognizable at a glance)

Reply:
Yeah, I'd actually called it something like `KOMAKE` at first (not sure if that was it; never committed that) but then I looked at `KOMAKE:=ninja` and changed the name. :-P

Reply:
It could also be called something like `(KO)MAKE_BACKEND`, although that makes me think more of GNU Make vs BSD Make or whatever. But maybe `KO_BUILD_BACKEND` sounds better than system?

Reply:
Or just something like `KO_CMAKE_GENERATOR` since it is in fact intended for CMake, with that `KO_` prefix to differentiate it from CMake's built-in `CMAKE_GENERATOR` thingy? Or possibly not even that because CMake doesn't care about environment variables like that, although that might be confusing.

Reply:
Well, except CMake uses names like `Ninja` and `Unix Makefiles` for that one. They *do* use `MAKE` in this one:

```
//Program used to build from build.ninja files.
CMAKE_MAKE_PROGRAM:FILEPATH=/usr/bin/ninja
```

So… `(KO_)CMAKE_MAKE_PROGRAM`?

Reply:
Yep, I like that one ;)!